scigym/envs/tests/README.md

```diff
@@ -0,0 +1,10 @@
+#### Unit Tests
+
+This folder contains unit tests for scigym environments, adapted from the openAI gym suite of tests.
+
+Run these tests by exececuting the following commands from within this directory:
+
+```python
+pytest test_envs.py test_determinism.py
```

Comment:
You can also run `python -m pytest` in terminal. The -m flag lets python know that you are using a python module.